### PR TITLE
A FOSS implementation of Delta

### PR DESCRIPTION
**Pull request by Donnnno**
It was actually pretty simple to implement

This should make it F-Droid compatible

It has two different flavors: play & foss. They both have their own library:, candybar and candybar-foss

I replaced the donate function with links for the foss version, other than that, you shouldn't notice any differences.

image

### CODE REVIEW

**Comment by Donnnno** [comment]
Oh, I had some weird building errors that it wanted me to downgrade to build tools 7.2.1 

and it had some duplicate things in dashboard_icon_pack.xml

**Comment by Donnnno** [comment]
Used an outdated android studio version, oops

**Comment by Donnnno** [comment]
Also, no need to change the Creative Commons license: https://f-droid.org/en/docs/FAQ_-_App_Developers/#how-do-i-license-my-app

**Comment by Donnnno** [comment]
I think it's good to go, if there are any questions, ask away :-)

File: app/src/foss/java/website/leifs/delta/applications/CandyBar.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        configuration.setIncludeIconRequestToEmailBody(true);
+        configuration.setShowTabAllIcons(true);
+        configuration.setShadowEnabled(false);
+        configuration.setDashboardThemingEnabled(false);
+        configuration.setCategoryForTabAllIcons(new String[] {
+                "Google", "System", "Folders", "Calendar", "Alts", "#", "A", "B", "C", "D", "E", "F", "G", "H", "I",
+                "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"
+        });
+
+        DonationLink[] donationLinks = new DonationLink[]{
```
Comment (Donnnno) [inline app/src/foss/java/website/leifs/delta/applications/CandyBar.java:42]: This is the place for donation links :)

**Comment by KerstinMaur** [comment]
This is awesome! 👍

**Pull request merged**